src/compute/divide.ts

```diff
@@ -0,0 +1,42 @@
+import { Image } from '../Image';
+import { getOutputImage } from '../utils/getOutputImage';
+import { validateChannels } from '../utils/validators/validators';
+
+//The used options are the same as in multiply function.
+import { MultiplyOptions } from './multiply';
+
+/**
+ *
+ * Divides image pixels by a certain value.
+ * @param image - image to which division will be applied.
+ * @param value - Value by which each pixel will be divided.
+ * @param options - Multiply options
```

Comment:
Divide not multiply

Reply:
It should be, but the interface I use is from MultiplyOptions. In the parameters docs should I put that it is divide options anyway?

Reply:
You should not use MultiplyOptions.

Reply:
But if the options are the same as with multiply function, should I create a DivideOptions interface anyway? Or should I do it differently?

Reply:
It's fine to have some duplicated types.

If you don't want to duplicate types, you can always do something like this:
```ts
interface OperationOptionsBase {
  channels?: number[];
  out?: Image
}

interface  MultiplyOptions extends OperationOptionsBase {};
interface  DivideOptions extends OperationOptionsBase {};
```

Reply:
Yes, in general, 1 function with options = 1 interface